Automattic/wp-calypso
# Gutenboarding: update design screenshots

## Changes proposed in this Pull Request

We're updating screenshots now that the previews are based on clean starter sites. 

See: https://github.com/Automattic/view-design/issues/54

Should only be merged after D45183-code is deployed

## Testing instructions

Apply D45183-code

Head over to `/new` and check out the design thumbnails.

They should look like the corresponding previews.

Fixes #44018
